ethexe/cli/src/service.rs

```diff
@@ -53,6 +53,7 @@ pub struct Service {
     db: Database,
     observer: ethexe_observer::Observer,
     query: ethexe_observer::Query,
+    router_query: RouterQuery,
```

Comment:
can't you create router query from observer?

Reply:
I can but what's the point? It is already created in `Service::new()`